### ISSUE TITLE
When i create a table in  Word, open it with onlyOffice online preview and the table overlaps  

### ISSUE THREAD

**Issue by baoyuping** [ONLYOFFICE/DocumentServer#1372]
**Do you want to request a *feature* or report a *bug*?**
It is a bug  
**What is the current behavior?**
open it with onlyOffice online preview and the table overlaps
![image](https://user-images.githubusercontent.com/75818314/128305830-3b68e4cd-a61d-4e73-a634-837c99672036.png)

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
1、 Create tables in Word using WPS or  Microsoft Office software
2、Use onlyOffice online preview to open it  
**What is the expected behavior?**
I expect the table in Word to display properly
**Did this work in previous versions of DocumentServer?**
No, it is working on the latest DocumentServer, version 6.3.1  
**DocumentServer version:**
 6.3.1  
**Operating System:**
win10
**Browser version:**
Google Chrome 91.0.4472.124

This is the document in question
[新建 DOCX 文档.docx](https://github.com/ONLYOFFICE/DocumentServer/files/6936711/DOCX.docx)

**Comment by ShockwaveNN** [comment]
@TatianaKirillova please take a look

**Comment by TatianaKirillova** [comment]
Hello, **@baoyuping**
Thanks for your request! I can confirm - this is know bug, issue 20296 in our private issue tracker.

**Comment by ShockwaveNN** [comment]
Possible duplicate of https://github.com/ONLYOFFICE/DocumentServer/issues/787 but I'm not sure, so leave this issue separate

**Comment by baoyuping** [comment]
you're welcome @TatianaKirillova

**Comment by baoyuping** [comment]
sorry，I found another problem. When I open Word with table in OnlyOffice, the table will not be displayed completely  。
![image](https://user-images.githubusercontent.com/75818314/128464954-f2a0cd13-84c8-43dd-b6a9-dce92e271cfb.png)
This is the attachment for the test
[Uploading 新建 DOCX 文档.docx…]()

**Comment by baoyuping** [comment]
sorry，I found another problem. When I open Word with table in OnlyOffice, the table will not be displayed completely  。
![image](https://user-images.githubusercontent.com/75818314/128464954-f2a0cd13-84c8-43dd-b6a9-dce92e271cfb.png)
This is the attachment for the test
[Uploading 新建 DOCX 文档.docx…]()

**Comment by ShockwaveNN** [comment]
@baoyuping you didnt attach file, link are goint to nowhere

I think those issues may be unreleteade, so please create a new issue

**Comment by baoyuping** [comment]
ok

**Comment by Mannixhu** [comment]
meet similar issue @ShockwaveNN @TatianaKirillova 
![Table mismatch and overlap](https://github.com/ONLYOFFICE/DocumentServer/assets/16033328/bae94ed4-075c-4c53-9a32-8c301cb60e00)
[Table mismatch and overlap.docx](https://github.com/ONLYOFFICE/DocumentServer/files/14162002/Table.mismatch.and.overlap.docx)

**Comment by TatianaKirillova** [comment]
@Mannixhu Indeed, there is a problem, and the reason is still the same. The problem is related to wrapping around floating tables.
